test/e2e/playbook/provision.yaml

```diff
@@ -21,28 +21,61 @@
         - sed -i 's/addresses\(.\) \[.*\]$/addresses\1 \[{{ dns_nameserver }}\]/' /etc/netplan/01-netcfg.yaml
       when: ansible_facts['distribution'] == "Ubuntu"
 
+    - name: disable swap
+      shell: "{{ item }}"
+      with_items:
+        - for swp in `systemctl --type swap | awk '/\.swap/ { print $1 }'`; do systemctl stop "$swp"; systemctl mask "$swp"; done
+        - swapoff --all
+      when: ansible_swaptotal_mb > 0
+
+    - name: download public signing key Kubernetes
+      apt_key:
+        url: https://packages.cloud.google.com/apt/doc/apt-key.gpg
+        state: present
+      when: ansible_facts['distribution'] == "Ubuntu"
+
+    - name: adding apt repository for Kubernetes
+      apt_repository:
+        repo: "deb https://apt.kubernetes.io/ kubernetes-xenial main"
+        state: present
+        filename: /etc/apt/sources.list.d/kubernetes.list
+      when: ansible_facts['distribution'] == "Ubuntu"
+
+    - name: adding yum repository for Kubernetes
+      yum_repository:
+        description: Kubernetes repository
+        baseurl: "https://packages.cloud.google.com/yum/repos/kubernetes-el7-$basearch"
+        gpgkey: "https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg"
+        state: present
+        name: kubernetes
+      when: ansible_facts['distribution'] == "Fedora"
+
+    - name: install common packages
+      package:
+        name:
+          - curl
+          - runc
+          - jq
+          - python3-pip
+          - kubelet
+          - kubeadm
+          - kubectl
+        state: present
+
```

Comment:
This is not related to this PR, but to ansible in general. Is it possible, in principle, to later add a condition here, if necessary, to this effect with some syntax ?
```
    when: ansible_facts['distribution'] == "Fedora" or ansible_facts['distribution'] == "Ubuntu"
```
That could come handy later if we all support for other distros and some of those require either a different set of packages or packages with different names...

Reply:
Yes, that's possible and the syntax you added here is correct.